### PR TITLE
Fix Client/server connection

### PR DESCRIPTION
**Pull request by apupier**
Signed-off-by: Aurélien Pupier <apupier@redhat.com>

### CODE REVIEW

File: clients/eclipse/org.apache.camel.lsp.eclipse.client/src/org/apache/camel/lsp/eclipse/client/CamelLSPStreamConnectionProvider.java [hunk 1]
```diff
@@ -38,7 +38,7 @@ public CamelLSPStreamConnectionProvider() {
 
 	private static String computeWorkingDir() {
 		return System.getProperty("user.dir");
-	}
+	}
```
Comment (lhein) [inline clients/eclipse/org.apache.camel.lsp.eclipse.client/src/org/apache/camel/lsp/eclipse/client/CamelLSPStreamConnectionProvider.java]: ?

File: server/src/main/java/org/apache/camel/tools/lsp/internal/CamelLanguageServer.java [hunk 2]
```diff
@@ -81,16 +79,6 @@ public void exit() {
 		ServerCapabilities capabilities = new ServerCapabilities();
 		capabilities.setTextDocumentSync(TextDocumentSyncKind.Incremental);
 		capabilities.setCompletionProvider(new CompletionOptions(Boolean.TRUE, Arrays.asList(".","?","&")));
-		capabilities.setHoverProvider(Boolean.FALSE);
```
Comment (lhein) [inline server/src/main/java/org/apache/camel/tools/lsp/internal/CamelLanguageServer.java]: why removing it when u can keep it and just switch to TRUE whenever a feature is ready?

Comment (apupier) [reply server/src/main/java/org/apache/camel/tools/lsp/internal/CamelLanguageServer.java]: less code is better code ;-)
it is causing NPE with LSP4e for the ones with a complex type provided. So at least these ones need to be removed.
Several of these features won't be ready before a long time so I would prefer to not see them.

File: clients/eclipse/org.apache.camel.lsp.eclipse.client/src/org/apache/camel/lsp/eclipse/client/CamelLSPStreamConnectionProvider.java (hunk 1 above)
Comment (apupier) [reply clients/eclipse/org.apache.camel.lsp.eclipse.client/src/org/apache/camel/lsp/eclipse/client/CamelLSPStreamConnectionProvider.java]: argh :s

**Pull request merged**